index.js

```diff
@@ -91,7 +91,7 @@ async function run() {
     }
 
     const numberOfFailedTests = testOutputObject.numberOfFailedTests;
-    const testRunFailed = (numberOfFailedTests && numberOfFailedTests > 0);
+    const testRunFailed = (numberOfFailedTests !== undefined && numberOfFailedTests !== 0 && numberOfFailedTests > 0);
```

Comment:
why `numberOfFailedTests !== 0` when there's already `numberOfFailedTests > 0`?

Reply:
Oh right!

Reply:
I left that in before I realized I should explicitly check it is not `undefined`